src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -647,14 +647,34 @@ class RecipeMarkdownGenerator : Runnable {
                 }
 
                 if (recipes.isNotEmpty()) {
+                    appendLine("## Composite Recipes")
+                    appendLine()
+                    appendLine("_Recipes that include further recipes, often including the individual recipes below._")
```

Comment:
```suggestion
                    appendLine()
                    appendLine("_Recipes that include further recipes, often including the individual recipes below._")
                    appendLine()
```

Reply:
Ahh good catch -- in my IDE it worked fine but ya there should definitely be a newline and I suspect it would have broken on GitBook.